scipy/stats/_distn_infrastructure.py

```diff
@@ -2559,12 +2560,26 @@ def fit(self, data, *args, **kwds):
 
         >>> from scipy.stats import beta
         >>> a, b = 1., 2.
-        >>> x = beta.rvs(a, b, size=1000)
+        >>> x = beta.rvs(a, b, size=1000, random_state=80)
```

Comment:
Elsewhere in the docs there has been a push to use `default_rng` with a larger seed to encourage use of `Generator`s and avoid repeated use of the same seed.